src/components/Edit/NatureOfBusiness.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div id="nature-of-business-info">
+    <NatureOfBusinessShared
+      v-if="haveNaics"
+      :showErrors="false"
+      :naics="getCurrentNaics"
+      :naics-services="naicsServices"
```

Comment:
Not a bit deal but prop style is different than usual here.

This is what I did:
https://github.com/bcgov/business-create-ui/blob/main/src/components/Registration/NatureOfBusiness.vue#L5

Reply:
Updated!